src/expressionEditor.js

```diff
@@ -72,8 +72,8 @@ define([
             richTextOptions = {isExpression: true},
             form = options.mug.form;
         options = _.defaults(options, {
-            leftPlaceholder: "Hint: drag a question here.",
-            rightPlaceholder: "Hint: drag a question here.",
+            leftPlaceholder: "Drag question here",
+            rightPlaceholder: "Drag question here",
```

Comment:
Should be made consistent with https://github.com/dimagi/Vellum/pull/590/files#diff-b860087dfb0ae7f09a955a95f517a309R333, which reads "Drag a question here"